### DISCUSSION TITLE
Illuminating emerald

### DISCUSSION THREAD

**Discussion by sadellie** [sadellie/unitto#23] [Announcements]
<p align="middle">
    <img src="https://user-images.githubusercontent.com/10384356/222153832-7c7f05e2-85c0-4161-baf4-57d756083bc8.png" width="99%" />
</p>

# Changes

- **Calculator.** It's fully functional, like everything works and you can replace your current calculator app with ***Unitto***.
  - Drag down for **history view**. Drag a little bit down to see latest item, drag more to reveal full list.
  - Use **cursor** to select, copy, paste and cut expressions.
  - Switch between **Radian** and **Degree** modes.
  - Use various **functions**: sin, arcsin, cos, modulo and etc.
- New setting. Choose **starting screen**: calculator or unit converter.
- Added **Nautical mile**.
- Updates are available on **F-Droid** again.

https://user-images.githubusercontent.com/10384356/222155224-8d9eed48-e329-4300-ac63-66f70d2affa4.mp4

Photo by <a href="https://unsplash.com/@aleksandrcc?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Aleksandr Buynitskiy</a> on <a href="https://unsplash.com/photos/cOiEJ9aupbU?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>

<hr /><em>This discussion was created from the release <a href='https://github.com/sadellie/unitto/releases/tag/illuminating-emerald'>Illuminating emerald</a>.</em>

**Comment by T8RIN** [comment]
Вопрос такой, а где можно такие классные картинки делать?

**Reply by sadellie** [reply]
Figma

**Reply by T8RIN** [reply]
Там какой-то набор М3 задников есть?

**Reply by T8RIN** [reply]
Хочу просто тоже такие фоны крутые, просто в интернете днем с огнем не сыщешь

**Reply by sadellie** [reply]
> Там какой-то набор М3 задников есть?

Нет, всё своё, создано с нуля.

**Reply by T8RIN** [reply]
Вау, шейпы тоже?

**Reply by sadellie** [reply]
> Вау, шейпы тоже?

Да
